core/src/main/scala/akka/kafka/ConsumerMessage.scala

```diff
@@ -181,7 +181,7 @@ object ConsumerMessage {
    * Java API:
    * Create an offset batch out of a list of offsets.
    */
-  def createCommittableOffsetBatch(offsets: java.util.List[CommittableOffset]): CommittableOffsetBatch = {
+  def createCommittableOffsetBatch[T <: Committable](offsets: java.util.List[T]): CommittableOffsetBatch = {
```

Comment:
The test I added proves that the type of this list has to be `T <: Committable` istead of `Committable`, but both types `Committable` and `T <: Committable` are binary-incompatible.

Reply:
Agreed. This seems fine to me. It's binary-incompatible, but since it's only widening permitted types it shouldn't break anything that was using the previous method.